apps/demos/Demos/CardView/CardTemplate/jQuery/index.js

```diff
@@ -1,7 +1,157 @@
 $(() => {
-  $('#card-view').dxCardView({
-    dataSource: customers,
-    keyExpr: 'ID',
-    columns: ['CompanyName', 'City', 'State', 'Phone', 'Fax'],
+  const popup = $('#popup').dxPopup({
+    width: 350,
+    height: 240,
+    visible: false,
+    dragEnabled: false,
+    hideOnOutsideClick: true,
+    title: 'Image Info',
+    position: {
+      at: 'center',
+      my: 'center',
+      collision: 'fit',
+    },
+  }).dxPopup('instance');
+
+  const createVehicleCard = (card) => {
+    const cardEl = $('<div>').addClass('vehicle__card');
+
+    const imageWrapper = createVehicleImg(card.data);
+    const vehicleInfo = createVehicleInfo(card);
+
+    cardEl.append(imageWrapper);
+    cardEl.append(vehicleInfo);
+
+    return cardEl;
+  };
+
+  const createVehicleImg = (vehicle) => {
+    const {
+      ID,
+      Name,
+      TrademarkName,
+    } = vehicle;
+    const imageWrapper = $('<div>').addClass('vehicle__img-wrapper');
+    const img = $('<img>').addClass('vehicle__img');
+    img.attr({
+      src: `../../../../images/vehicles/image_${ID}.png`,
+      alt: `${TrademarkName} ${Name}`,
+    });
+
+    imageWrapper.append(img);
+
+    return imageWrapper;
+  };
+
+  const createVehicleInfo = (card) => {
+    const {
+      TrademarkName,
+      Name,
+      CategoryName,
+      Modification,
+      BodyStyleName,
+      Horsepower,
+      Source,
+      LicenseName,
+      Author,
+      Edits,
+    } = card.data;
+    const vehicleInfo = $('<div>').addClass('vehicle__info');
+
+    const name = `${TrademarkName} ${Name}`;
+    const vehicleName = $('<div>').addClass('vehicle__name').text(name).attr('title', name);
+
+    const priceText = card.fields.find(f => f.column.dataField === 'Price')?.text;
+    const priceInfo = $('<div>').addClass('vehicle__price').text(`${priceText}`);
+
+    const typeContainer = $('<div>').addClass('vehicle__type-container');
+    const typeInfo = $('<div>').addClass('vehicle__type').text(`${CategoryName}`);
```

Comment:
What is the point of this string interpolation? :)

```suggestion
    const typeInfo = $('<div>').addClass('vehicle__type').text(CategoryName);
```